### PR TITLE
added .json loader to webpack.config.js in template to make aws-sdk w…

### PR DESCRIPTION
**Pull request by stormlrd**
added .json loader to webpack.config.js in template to make aws-sdk work when using var AWS=require('aws-sdk'); in the slackbot.js handler

spent ages working around an error about loader and json files when my slack bot project had aws-sdk in it and trying to serverless deploy using serverless 1.4.0 

skipping parsing the sdk doesn't help. this worked. so figured it was worth a pull.

### CODE REVIEW

File: template/webpack.config.js [hunk 1]
```diff
@@ -8,6 +8,11 @@ module.exports = {
         loaders: ['babel'],
         exclude: /node_modules/,
       }
+    ],
```
Comment (kddeisz) [inline template/webpack.config.js]: ✂️

**Comment by kddeisz** [comment]
Thanks @stormlrd! Could you fix the syntax in the JSON file and add the `json-loader` package to the devDependencies in `package.json`? Btw, you can test the template with `bin/test` (what travis is running).

**Comment by stormlrd** [comment]
Done as requested. Sorry for not being tidier. Was a late night! :)

**Comment by kddeisz** [comment]
No problem! Last thing is just to remove the `npm-debug.log` file

**Pull request merged**